typescript/infra/hardhat.config.ts

```diff
@@ -29,7 +29,7 @@ const chainSummary = async <Chain extends ChainName>(
   const inboxSummary = async (remote: Chain) => {
     const inbox = coreContracts.inboxes[remote as Exclude<Chain, Chain>].inbox;
     const [inboxCheckpointRoot, inboxCheckpointIndex] =
-      await inbox.latestCheckpoint();
+      await inbox.latestCachedCheckpoint();
```

Comment:
We should probably just remove this from the summary, doesn't seem that useful anymore